modules/sync_staging_table_with_production.nf

```diff
@@ -0,0 +1,18 @@
+// Create data guide
+process sync_staging_table_with_production {
+    debug true
+    container "$params.sync_table_docker"
+    secret 'SYNAPSE_AUTH_TOKEN'
+
+    input:
+    val is_staging
```

Comment:
Just realized but can you just remove this `input` block because we don't need to use `is_staging` here?